fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1174,6 +1178,57 @@ public PlanFragment visitPhysicalPaimonScan(OptExpression optExpression, ExecPla
             return fragment;
         }
 
+        @Override
+        public PlanFragment visitPhysicalOdpsScan(OptExpression optExpression, ExecPlan context) {
+            LOG.info("[debug] start visit physical odps scan");
```

Comment:
please remove this debug log.

Reply:
@zdx19981006 will fix it in the next comment